client/src/components/all-components/GettingInvolved.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ALLButton from "./ALLButton";
+import { navigate } from "@reach/router";
+const GettingInvolved = () => {
+  const handleNav = () => {
+    navigate("/#contact");
+  };
+
+  return (
+    <div className={"tw-h-[25rem] tw-w-full"}>
+      <div
+        className={
+          "tw-h-1/2 tw-border-t-[4rem] tw-border-r-[1.5rem] tw-border-b-0 tw-border-l-0 tw-border-solid tw-border-primary-yellow tw-bg-primary-yellow"
+        }
+      >
+        <div
+          className={
+            "tw-h-full tw-flex tw-flex-row tw-justify-center tw-align-middle tw-border-t-[.75rem] tw-border-r-[.75rem] tw-rounded-tr-lg tw-border-b-0 tw-border-l-0 tw-border-solid tw-border-primary-blue tw-bg-white"
+          }
+        >
+          <div
+            className={`tw-flex tw-flex-row tw-text-left sm:tw-w-full lg:tw-w-1/2`}
+          >
+            <div
+              className={
+                "tw-flex tw-flex-col tw-w-full tw-h-full tw-justify-center tw-m-[3rem]"
+              }
+            >
+              <p
+                className={
+                  "tw-title-styling-name xxs:tw-text-sm md:lg:tw-text-2xl tw-my-3 tw-font-poppins md:lg:tw-w-1/2"
+                }
+              >
+                {" "}
+                Interested in Getting Involved?
+              </p>
+              <div className={"tw-flex tw-flex-row  tw-items-center"}>
+                <p
+                  className={
+                    "tw-font-calibri tw-font-medium sm:tw-text-sm xs:tw-text-xs"
+                  }
+                >
+                  {" "}
+                  Whether you want to implement our labs in your course
+                  curriculum, join the Accessible Learning Labs development
+                  partners or anything in between, you can click here to learn
+                  more!
```

Comment:
![Screenshot 2024-10-24 at 12 40 01 PM](https://github.com/user-attachments/assets/d632dd9a-1a0c-4266-9892-d7b39b449eb9)

The font size for this body copy is too small. Body copy should be 18px according to the style guide, or 1.125 rem